index.html

```diff
@@ -71,7 +72,7 @@ <h1 class="main-head">
                 us.
               </p>
               <button class="big-btn-hsection">Get Started</button>
-              <div class="decoration-shadow"></div>
+              <div class="decoration-shadow" aria-hidden="true"></div>
```

Comment:
There is no need to add aria-hidden attribute for the empty element.

Reply:
Ok, understood.